applications/FreeSurfaceApplication/python_scripts/free_surface_analysis.py

```diff
@@ -14,12 +15,42 @@ def __init__(self,
         # - self.project_parameters
         # - self.echo_level
         # - self.parallel_type
-        AnalysisStage.__init__(self, model, parameters)
+        super(FreeSurfaceAnalysis,self).__init__(model,parameters)
 
+    def Initialize(self):
+        super(FreeSurfaceAnalysis,self).Initialize()
```

Comment:
```suggestion
        super().Initialize()
```
As we dropped Python2 support you can directly call the `super` like this.

Reply:
Done